utils/sip-dig.c

```diff
@@ -223,7 +223,7 @@ int main(int argc, char *argv[])
 
   while (argv[1] && argv[1][0] == '-') {
     if (strcmp(argv[1], "-v") == 0)
```

Comment:
Is it safe to remove the `if`?

Reply:
And what about line 86?

Reply:
didn't want to remove `if` completely, because it can be useful in the future if we would like this command to have really working verbose option

Reply:
So it is safe. Then let's remove what's declared as it is not functional and confuses as a feature that does not exist.

Reply:
removed